### ISSUE TITLE
YOLOv9 is better than any convolution or transformer based object detectors

### ISSUE THREAD

**Issue by AlexeyAB** [AlexeyAB/darknet#8887]
**YOLOv9** is better than any convolution or transformer based object detectors

- **Paper**: https://arxiv.org/abs/2402.13616
- **Code**: https://github.com/WongKinYiu/yolov9

![image](https://github.com/AlexeyAB/darknet/assets/4096485/c23859a3-785d-4679-b3da-bf7df9389ff5)

**Comment by waytechpro** [comment]
Absolutely! YOLOv9 certainly stands out among object detectors with its efficient architecture and remarkable performance. Its streamlined approach surpasses traditional convolutional and transformer-based models in terms of speed and accuracy. If you're keen on exploring cutting-edge object detection capabilities, YOLOv9 is definitely worth considering. Ready to elevate your detection game? [Click here](https://waytechpro.com/leads-360/) to learn more.

**Comment by 1027663760** [comment]
Have you considered adapting YOLOV9 for Darknet?

**Comment by toplinuxsir** [comment]
It is a great idea!

**Comment by TaihuLight** [comment]
It is better to implement YOLOV9 in this version of darknet rather than the newer one [hank-ai/darknet](https://github.com/hank-ai/darknet). This is because C programming is more suitable for edge devices than C++.

**Comment by stephanecharette** [comment]
>This is because C programming is more suitable for edge devices than C++.

As someone who uses Darknet/YOLO on edge devices, I strongly disagree.

**Comment by stephanecharette** [comment]
You may also want to watch this video:  https://www.youtube.com/watch?v=2Mq23LFv1aM
